dsp/window/window_example_test.go

```diff
@@ -0,0 +1,388 @@
+// Copyright ©2018 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package window_test
+
+import (
+	"fmt"
+	"math"
+	"math/cmplx"
+
+	"gonum.org/v1/gonum/dsp/window"
+	"gonum.org/v1/gonum/fourier"
+)
+
+func Example() {
+	// The input sequence is a Sin function with 2.5 pereod per all sequence length
```

Comment:
s/pereod/period/

I'm not entirely sure what "per all sequence length" means.

Final full stop.

Reply:
The input sequence is a 2.5 period of the Sin function?
What do you mean by "Final full stop".

Reply:
OK. "per all sequence length is unclear".

Please use a full stop (period) at the end of the sentence.